schema/establishment.js

```diff
@@ -6,6 +6,12 @@ class Establishment extends BaseModel {
     return 'establishments';
   }
 
+  static get unlicensed() {
+    return [
+      'cjsmEmail'
```

Comment:
If we're going down this road...

```suggestion
      'billing',
      'cjsmEmail',
      'sharedKey'
```

Reply:
yeah I didn't definitely know